mteb/tasks/Retrieval/slk/SlovakSumRetrieval.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class SlovakSumRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="SlovakSumRetrieval",
+        description="""
+            SlovakSum, a Slovak news summarization dataset consisting of over 200 thousand
+            news articles with titles and short abstracts obtained from multiple Slovak newspapers.
+
+            Originally intended as a summarization task, but since no human annotations were provided
+            here reformulated to a retrieval task.
+        """,
+        reference="https://huggingface.co/datasets/kiviki/SlovakSum",
+        dataset={
+            "path": "kiviki/SlovakSum",
+            "revision": "85d6b32f2762313714618171b9d1a65eb7408835",
+        },
+        type="Retrieval",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["slk-Latn"],
+        main_score="ndcg_at_1",
```

Comment:
I think the main metric is ndcg@10

Reply:
yeah, but I wonder if that makes sence here in that instace, as here we have only 1 relevant document, wdyt?

Reply:
Of I'm not mistaking, the k is for the rank, so here with ndcg@1 you only look at the firstly ranked document and see if it's the relevant one. But with ndcg@10 you look at top-10 documents and expect the relevant one to be among them.

Reply:
Talking about the main metric, I'm not even sure it's ndcg, I think that for other tasks we put MAP `map`, can you please check so that we'd be consistent?

Reply:
So here's how it looks for all retrieval tasks:

```
         'ndcg_at_10': 89,
         'ndcg_at_20': 2,
         'map': 2,
         'mrr_at_5': 1,
         'ndcg_at_100': 1,
         'ndcg_at_1': 1,
         'ncdg_at_10': 1,
```

so indeed I change to `ndcg_at_10`.

Reply:
BTW `ncdg_at_10` looks like a type, mind if as a scout rule I fix in this PR?

Reply:
Yep it's a typo you can fix it yes

Reply:
Done ✅